salt/reviewer-suggestions/init.sls

```diff
@@ -25,13 +25,25 @@ reviewer-suggestions-build-essential:
         - pkgs:
             - build-essential
 
+reviewer-suggestions-server-service-stopped:
+    service.dead:
+        - onlyif: ls /etc/init/reviewer-suggestions-server.conf
+        - name: reviewer-suggestions-server
+
+reviewer-suggestions-server-service-started:
+    service.running:
+        - order: last
+        - name: reviewer-suggestions-server
+        - require:
```

Comment:
I think that `require` and `order` are alternative: either use `order` to put it at the end or make sure `require` is enough to establish the dependency order